courses/machine_learning/deepdive2/structured/labs/5a_train_keras_ai_platform_babyweight.ipynb

```diff
@@ -0,0 +1,749 @@
+{
```

Comment:
<p>maybe add a short reminder that they created this dataset in 1b_prepare_data_babyweight.ipynb <a href="https://github.com/GoogleCloudPlatform/training-data-analyst/blob/master/courses/machine_learning/deepdive2/structured/labs/1b_prepare_data_babyweight.ipynb" target="_blank">https://github.com/GoogleCloudPlatform/training-data-analyst/blob/master/courses/machine_learning/deepdive2/structured/labs/1b_prepare_data_babyweight.ipynb</a></p><p>and if the data doesn't exist, then redo that lab or run the solution.</p><p> </p><p>(adjust solution lab accordingly)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/training-data-analyst/pull/674/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='10'/>

Reply:
<p>Good idea, I did that in the BQML labs. I added here to here too now.</p>